runtime-sdk/src/modules/rofl/test.rs

```diff
@@ -242,6 +242,70 @@ fn test_create_scheme() {
     );
 }
 
+#[test]
+fn test_derive_app_key_id() {
+    // Ensure that app key identifier derivation is stable.
+    let tcs = [
```

Comment:
Named fields are more readable, but this is shorter. I guess both solutions are fine :thinking: 

```rust
    struct TestCase {
        key: &'static str,
        kind: types::KeyKind,
        scope: types::KeyScope,
        data: &'static [u8],
        registration: Option<types::Registration>,
        expected: &'static str,
    }
```